### ISSUE TITLE
Installation fails on missing 'logs' file directory I think

### ISSUE THREAD

**Issue by sgoggins** [chaoss/augur#785]
Please help us help you by filling out the following sections as thoroughly as you can.

**Description:**
```
GitLab API Key: xxxx-xxxx

The Facade data collection worker will clone repositories to this machine to run its analysis.
Would you like to clone to an existing directory, or create a new one?
1) Use an existing directory
2) Create a new directory

Please type the number corresponding to your selection and then press the Enter/Return key.
Your choice: /mnt/md0/repos/xxxx-repos-2/

Please type the number corresponding to your selection and then press the Enter/Return key.
Your choice: 1
** You MUST use an absolute path. Variable expansion is currently not supported.**
Facade repo path: /mnt/md0/repos/xxxx-repos-2/


**********************************
Generating configuration file...
**********************************

[INFO] Attempting to load config file
[WARNING] Config file not found. Using default configuration
CLI: [configure.generate] [INFO] augur.xxxx.json successfully created
[INFO] Attempting to load config file
[INFO] Config file loaded successfully
CLI: [db.check_pgpass_credentials] [INFO] Credentials found in $HOME/.xxx
Would you like to install Augur's frontend dependencies? [Y/n] y
scripts/install/frontend.sh: line 7: logs/install/frontend.log: No such file or directory
make: *** [Makefile:40: install-dev] Error 1
(augur-xxxx) sean@xxxx:~/github/augur-xxxx$ 
```
Latest version of `dev` ... pretty sure its just an anomaly not covered because I'm reinstalling

**Comment by sgoggins** [comment]
perhaps a `mkdir logs/install` is not executed under some conditions.

**Comment by ccarterlandis** [comment]
I just checked, it used to be created. I must have deleted those lines by accident, for now you can create it manually and it will work fine. I’ll add it back in later @sgoggins

**Comment by sgoggins** [comment]
OK ... it erases the logs directory when I do a `make rebuild-dev` also.

**Comment by ccarterlandis** [comment]
That’s weird, not sure why it would do that. I’ll take a look into that as well

**Comment by ccarterlandis** [comment]
Fixed in `dev` with PR #786
